ui/app/AppLayouts/Chat/views/ChatContentView.qml

```diff
@@ -454,8 +455,7 @@ ColumnLayout {
             Layout.alignment: Qt.AlignHCenter | Qt.AlignBottom
             Layout.fillWidth: true
             Layout.preferredWidth: parent.width
-            height: chatInput.height
-            Layout.preferredHeight: height
+            Layout.preferredHeight: chatInput.implicitHeight + chatInput.anchors.leftMargin + chatInput.anchors.rightMargin
```

Comment:
Looks wrong that height is obtained using left and right margin (instead of top and bottom) :thinking: